### ISSUE TITLE
Rename "Supernatural" attribute category to "Extraordinary"

### ISSUE THREAD

**Issue by brianfeister** [openlegend/core-rules#161]
The motivation is that some Super hero powers are technically considered science and not supernatural. Also, the far future settings where attributes are expressed through technology have trouble with the term "Supernatural" as well.

Let's discuss.

**Comment by DMsShadow** [comment]
From my man flu deathbed I can say I like it. While it could be supernatural, it leaves it nice and generic for those other settings.

**Comment by DukeAnax** [comment]
makes a lot of sense and I see no issues.

**Comment by istabosz** [comment]
I like it. I like it a lot.

**Comment by Odog4ever** [comment]
No reason not to change it.

**Comment by EliManthey** [comment]
I think it is more applicable to different genres...

**Comment by OneEyeRob** [comment]
Yeah, i completely agree.

**Comment by ucffool** [comment]
@brianfeister @istabosz If you guys want to call it, I'm happy to work on updating and committing the change from Supernatural to Extraordinary through the repo.

**Comment by brianfeister** [comment]
Thanks @ucffool. I'm drowning in stuff to do right now. That would great if you want to tackle it, thanks!

**Comment by ucffool** [comment]
@brianfeister [Committed](https://github.com/openlegend/core-rules/pull/183/commits/e6d0552278a29d9a97f3a4dae3c44ed5fe3498b4) and thus automatically wrapped into Pull Request #183

**Comment by Elvenoob** [comment]
I'm not sure about some of those changes @ucffool , the ones referring to the mechanics are fine, but the changes to the flavour text of certain perks, Supernatural Presence and Divine Insight, make no sense because those perks are inherently magical in other parts of their descriptions.

**Comment by ucffool** [comment]
@Elvenoob I looked over Supernatural Presence (now Extraordinary Presence) and considered that in the changed state, the character of sentient robot (Cyberpunk or DC Comics) could better fit it to them, which is part of the reason for the change. If it was back to Supernatural, it would exclude that world-type.

Regarding Divine Insight, I agree, I think it pushes it too far, so I've changed it back to supernatural within the description.

Also, found a typo (tred instead of tread) in Extraordinary Presence because of the second look, so thanks. :)
